src/components/JsonLink.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Icon } from './Icon.js';
+import Colors from './Colors.js';
+
+export default function JsonLink(props) {
+  const handleClick = (e) => {
+    e.preventDefault();
+    window.open(`data:application/json, ${encodeURIComponent(JSON.stringify(props.json, null, 1))}`, '_blank');
```

Comment:
Then this appears to assume props.json is an object even though propTypes specifies a string right now.

I think this bit should work with either a string or an object.